gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -444,8 +456,19 @@ public void createBucket(String bucketName, CreateBucketOptions options) throws
     Bucket bucket =
         new Bucket()
             .setName(bucketName)
+            .setHierarchicalNamespace(
```

Comment:
This is the createBucket flow. We do not support creating HN bucket from connector since we wont know if customer expect to create an HN bucket or non-HN bucket. But we need it for testing purposed. This change is just for testing since it need to create an HN bucket to test.

Reply:
one way of removing any impact on existing customer is to put it under `options.getHierarchicalNamespaceEnabled()` for eg:
```
Bucket bucket =
        new Bucket()
            .setName(bucketName)
            .setLocation(options.getLocation())
            .setStorageClass(options.getStorageClass());
if(options.getHierarchicalNamespaceEnabled()) {
    bucket =
        new Bucket()
            .setName(bucketName)
            .setLocation(options.getLocation())
            .setHierarchicalNamespace(.....
            .setStorageClass(options.getStorageClass());
    
    bucket.setIamConfiguration(
          new Bucket.IamConfiguration()
              .setUniformBucketLevelAccess(
                  new Bucket.IamConfiguration.UniformBucketLevelAccess().setEnabled(true)));
}
```

Reply:
as discussed offline this will be fixed in follow up PR cc: @arunkumarchacko